src/configuration.ts

```diff
@@ -128,15 +129,46 @@ export function loadConfigurationFromPath(configFilePath: string): IConfiguratio
         console.warn(PACKAGE_DEPRECATION_MSG);
         return require(configFilePath).tslintConfig;
     } else {
-        let fileData = fs.readFileSync(configFilePath, "utf8");
-        // remove BOM if present
-        fileData = fileData.replace(/^\uFEFF/, "");
-        const configFile: IConfigurationFile = JSON.parse(fileData);
-        configFile.rulesDirectory = getRulesDirectories(configFile.rulesDirectory, path.dirname(configFilePath));
+        let configFile: IConfigurationFile;
```

Comment:
Then with the changes noted below, we should be able to do something like:
```
const resolvedConfigFilePath = resolveConfigurationPath(configFilePath);
const configFile: IConfigurationFile = require(resolvedConfigFilePath);
const configFileDir = path.dirname(resolvedConfigFilePath);
```

We should be able to eliminate lines 132 through 143 then